tests/manage/test_pvc_disruptive.py

```diff
@@ -117,47 +263,55 @@ class TestFSDisruption(BaseDisruption):
     """
     CephFS PVC related disruption tests class
     """
-    storage_type = 'fs'
+    interface = constants.CEPHFILESYSTEM
```

Comment:
I think run_io expects either 'fs' or 'block'

Reply:
Yes. storage_type='fs' is passed while calling run_io.

Reply:
right :)